src/components/items/stars/empty.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It probably makes sense to put this, and the other svgs, into a `/assets` or `/icons` file in the src directory.

Reply:
added in 4ac62df74c5bdab5927a623d702adcd0e14dafe6